calour/sorting.py

```diff
@@ -43,6 +44,48 @@ def sort_taxonomy(exp, inplace=False):
 
 
 @Experiment._record_sig
+def sort_center_mass(exp, transform=log_n, inplace=False, **kwargs):
+    '''Sort the features based on the center of mass
+
+    Assumes exp samples are sorted by some continuous field,
+    and for each feature finds the center of mass (i.e. value * position along field).
```

Comment:
could you write complete sentence? it is not syntactically correct. And `value * position along field` is not understandable.